### PR TITLE
TAP 1.3.0 release updates

### PR DESCRIPTION
**Pull request by satya-dillikar**
*Issue #, if available:*

*Description of changes:*

- Update TAP install scripts for TAP new release 1.3.0
- Update AcceptEULAs & AcceptCEIP ParameterLabels Descriptions.
- Add default value for EKS cluster name
- Update TAP 1.3 docs references
- Added TAP profile "testing_scanning"
-  Update Scan policy
- Integrate metadata store for TAP-GUI

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tap-setup-scripts/src/inputs/user-input-values.example.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ tap:
   name: tap
   namespace: tap-install
   repository: registry.tanzu.vmware.com/tanzu-application-platform/tap-packages
-  version: 1.2.0
+  version: 1.3.0-rc.1
```
Comment (satya-dillikar) [inline tap-setup-scripts/src/inputs/user-input-values.example.yaml:16]: this is just a sample example file. The values are for illustration purposes only. not actually used.

File: tap-setup-scripts/src/functions.sh [hunk 2]
```diff
@@ -186,7 +186,6 @@ function readUserInputs {
   TAP_ECR_REGISTRY_REPOSITORY=$(yq -r .repositories.tap_packages $INPUTS/user-input-values.yaml)
   ESSENTIALS_ECR_REGISTRY_REPOSITORY=$(yq -r .repositories.cluster_essentials $INPUTS/user-input-values.yaml)
   TBS_ECR_REGISTRY_REPOSITORY=$(yq -r .repositories.build_service $INPUTS/user-input-values.yaml)
-  TBS_IAM_ROLE_ARN=$(yq -r .build_service.buildservice_arn $INPUTS/user-input-values.yaml)
```
Comment (satya-dillikar) [inline tap-setup-scripts/src/functions.sh:189]: unused variable

File: tap-setup-scripts/src/inputs/tap-values.yaml [hunk 3]
```diff
@@ -5,22 +5,34 @@ ceip_policy_disclosed: true # Installation fails if this is set to 'false'
 buildservice:
   kp_default_repository: #@ data.values.repositories.build_service
   kp_default_repository_aws_iam_role_arn: #@ data.values.buildservice.buildservice_arn
-  tanzunet_username: #@ data.values.tanzunet.username
```
Comment (satya-dillikar) [inline tap-setup-scripts/src/inputs/tap-values.yaml:8]: unused variables, hence removed

Comment (tlindsay42) [reply tap-setup-scripts/src/inputs/tap-values.yaml:8]: Thanks for the additional context, @satya-dillikar. 😃

**Comment by satya-dillikar** [comment]
> end to end tests passed. [taskcat report](https://vmware-tanzu-application-platform-buildlogs.s3.us-west-2.amazonaws.com/pr/29/index.html)

Thanks for taking care of this. I am unable to access the taskcat report s3 URL.  @tlindsay42, please copy-paste the taskcat report here.

**Pull request merged**